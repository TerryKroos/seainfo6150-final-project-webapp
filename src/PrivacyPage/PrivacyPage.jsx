import React from 'react';
import NavBar from '../NavBar/NavBar.jsx';
import Footer from '../Footer/Footer.jsx'
import styles from './PrivacyPage.module.css';


const PrivacyPage = () => {
    return(
        <div>
           <NavBar/>
           <div className={styles.container}>
              <div className={styles.flexcontainer}>
                 <h1>Privacy Policy</h1>
                 <h3>What information do we collect?</h3>
                 <p>When you register as a user of this site, we may collect your email address, name and address, telephone number, and payment information (e.g., credit card number and billing address). Additionally, when you visit this site, we automatically collect some non-personally identifiable computer information, such as the type of computer operating system (such as Windows XP or Mac OS) and web browser (such as Chrome or Internet Explorer) you are using, information regarding your Internet service provider, and the frequency with which you visit various parts of this site. We use this information to manage the Website and make it more interesting and useful to you. We may offer public commenting through this site. If you use our commenting system and disclose personally identifiable information, you should be aware that such information could be read, collected or used by other users. We are not responsible for the personally identifiable information you choose to submit through public forums.</p>
                 <h3>What information do we collect?</h3>
                 <p>When you register as a user of this site, we may collect your email address, name and address, telephone number, and payment information (e.g., credit card number and billing address). Additionally, when you visit this site, we automatically collect some non-personally identifiable computer information, such as the type of computer operating system (such as Windows XP or Mac OS) and web browser (such as Chrome or Internet Explorer) you are using, information regarding your Internet service provider, and the frequency with which you visit various parts of this site. We use this information to manage the Website and make it more interesting and useful to you. We may offer public commenting through this site. If you use our commenting system and disclose personally identifiable information, you should be aware that such information could be read, collected or used by other users. We are not responsible for the personally identifiable information you choose to submit through public forums.</p>
                 <h3>What information do we collect?</h3>
                 <p>When you register as a user of this site, we may collect your email address, name and address, telephone number, and payment information (e.g., credit card number and billing address). Additionally, when you visit this site, we automatically collect some non-personally identifiable computer information, such as the type of computer operating system (such as Windows XP or Mac OS) and web browser (such as Chrome or Internet Explorer) you are using, information regarding your Internet service provider, and the frequency with which you visit various parts of this site. We use this information to manage the Website and make it more interesting and useful to you. We may offer public commenting through this site. If you use our commenting system and disclose personally identifiable information, you should be aware that such information could be read, collected or used by other users. We are not responsible for the personally identifiable information you choose to submit through public forums.</p>
                 <h3>What information do we collect?</h3>
                 <p>When you register as a user of this site, we may collect your email address, name and address, telephone number, and payment information (e.g., credit card number and billing address). Additionally, when you visit this site, we automatically collect some non-personally identifiable computer information, such as the type of computer operating system (such as Windows XP or Mac OS) and web browser (such as Chrome or Internet Explorer) you are using, information regarding your Internet service provider, and the frequency with which you visit various parts of this site. We use this information to manage the Website and make it more interesting and useful to you. We may offer public commenting through this site. If you use our commenting system and disclose personally identifiable information, you should be aware that such information could be read, collected or used by other users. We are not responsible for the personally identifiable information you choose to submit through public forums.</p>
                 <h3>What information do we collect?</h3>
                 <p>When you register as a user of this site, we may collect your email address, name and address, telephone number, and payment information (e.g., credit card number and billing address). Additionally, when you visit this site, we automatically collect some non-personally identifiable computer information, such as the type of computer operating system (such as Windows XP or Mac OS) and web browser (such as Chrome or Internet Explorer) you are using, information regarding your Internet service provider, and the frequency with which you visit various parts of this site. We use this information to manage the Website and make it more interesting and useful to you. We may offer public commenting through this site. If you use our commenting system and disclose personally identifiable information, you should be aware that such information could be read, collected or used by other users. We are not responsible for the personally identifiable information you choose to submit through public forums.</p>
                 <h3>What information do we collect?</h3>
                 <p>When you register as a user of this site, we may collect your email address, name and address, telephone number, and payment information (e.g., credit card number and billing address). Additionally, when you visit this site, we automatically collect some non-personally identifiable computer information, such as the type of computer operating system (such as Windows XP or Mac OS) and web browser (such as Chrome or Internet Explorer) you are using, information regarding your Internet service provider, and the frequency with which you visit various parts of this site. We use this information to manage the Website and make it more interesting and useful to you. We may offer public commenting through this site. If you use our commenting system and disclose personally identifiable information, you should be aware that such information could be read, collected or used by other users. We are not responsible for the personally identifiable information you choose to submit through public forums.</p>
              </div>
           </div>
           <Footer/>
        </div>
        );
}

export default PrivacyPage;