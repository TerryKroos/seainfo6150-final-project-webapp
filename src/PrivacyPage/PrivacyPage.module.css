.container {
    display: flex;
    align-items: center;
    justify-content: center;
    font-family: 'Inter', sans-serif;
}

.flexcontainer{
    width: 60em;
}