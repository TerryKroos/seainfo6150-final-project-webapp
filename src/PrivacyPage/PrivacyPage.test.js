
import React from "react";
import PrivacyPage from "./PrivacyPage";
import { render } from "@testing-library/react";

describe("PrivacyPage tests", () => {
    it("renders correctly", () => {
      const { container } = render(<PrivacyPage />);
      expect(container).toMatchSnapshot();
    });
  
  });