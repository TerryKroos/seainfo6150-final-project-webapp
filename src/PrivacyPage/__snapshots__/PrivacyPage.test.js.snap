// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PrivacyPage tests renders correctly 1`] = `
<div>
  <div>
    <nav>
      <ul>
        <li>
          The Sports Blog
        </li>
        <li>
          <a
            class="link"
            href="/"
          >
            HOME
          </a>
        </li>
        <li>
          <a
            class="link"
            href="/nba"
          >
            NBA
          </a>
        </li>
        <li>
          <a
            class="link"
            href="/nfl"
          >
            NFL
          </a>
        </li>
        <li>
          <a
            class="link"
            href="/nhl"
          >
            NHL
          </a>
        </li>
        <li>
          <a
            class="link"
            href="/mbl"
          >
            MBL
          </a>
        </li>
        <li>
          <a
            class="link"
            href="/soccer"
          >
            SOCCER
          </a>
        </li>
      </ul>
    </nav>
    <div
      class="container"
    >
      <div
        class="flexcontainer"
      >
        <h1>
          Privacy Policy
        </h1>
        <h3>
          What information do we collect?
        </h3>
        <p>
          When you register as a user of this site, we may collect your email address, name and address, telephone number, and payment information (e.g., credit card number and billing address). Additionally, when you visit this site, we automatically collect some non-personally identifiable computer information, such as the type of computer operating system (such as Windows XP or Mac OS) and web browser (such as Chrome or Internet Explorer) you are using, information regarding your Internet service provider, and the frequency with which you visit various parts of this site. We use this information to manage the Website and make it more interesting and useful to you. We may offer public commenting through this site. If you use our commenting system and disclose personally identifiable information, you should be aware that such information could be read, collected or used by other users. We are not responsible for the personally identifiable information you choose to submit through public forums.
        </p>
        <h3>
          What information do we collect?
        </h3>
        <p>
          When you register as a user of this site, we may collect your email address, name and address, telephone number, and payment information (e.g., credit card number and billing address). Additionally, when you visit this site, we automatically collect some non-personally identifiable computer information, such as the type of computer operating system (such as Windows XP or Mac OS) and web browser (such as Chrome or Internet Explorer) you are using, information regarding your Internet service provider, and the frequency with which you visit various parts of this site. We use this information to manage the Website and make it more interesting and useful to you. We may offer public commenting through this site. If you use our commenting system and disclose personally identifiable information, you should be aware that such information could be read, collected or used by other users. We are not responsible for the personally identifiable information you choose to submit through public forums.
        </p>
        <h3>
          What information do we collect?
        </h3>
        <p>
          When you register as a user of this site, we may collect your email address, name and address, telephone number, and payment information (e.g., credit card number and billing address). Additionally, when you visit this site, we automatically collect some non-personally identifiable computer information, such as the type of computer operating system (such as Windows XP or Mac OS) and web browser (such as Chrome or Internet Explorer) you are using, information regarding your Internet service provider, and the frequency with which you visit various parts of this site. We use this information to manage the Website and make it more interesting and useful to you. We may offer public commenting through this site. If you use our commenting system and disclose personally identifiable information, you should be aware that such information could be read, collected or used by other users. We are not responsible for the personally identifiable information you choose to submit through public forums.
        </p>
        <h3>
          What information do we collect?
        </h3>
        <p>
          When you register as a user of this site, we may collect your email address, name and address, telephone number, and payment information (e.g., credit card number and billing address). Additionally, when you visit this site, we automatically collect some non-personally identifiable computer information, such as the type of computer operating system (such as Windows XP or Mac OS) and web browser (such as Chrome or Internet Explorer) you are using, information regarding your Internet service provider, and the frequency with which you visit various parts of this site. We use this information to manage the Website and make it more interesting and useful to you. We may offer public commenting through this site. If you use our commenting system and disclose personally identifiable information, you should be aware that such information could be read, collected or used by other users. We are not responsible for the personally identifiable information you choose to submit through public forums.
        </p>
        <h3>
          What information do we collect?
        </h3>
        <p>
          When you register as a user of this site, we may collect your email address, name and address, telephone number, and payment information (e.g., credit card number and billing address). Additionally, when you visit this site, we automatically collect some non-personally identifiable computer information, such as the type of computer operating system (such as Windows XP or Mac OS) and web browser (such as Chrome or Internet Explorer) you are using, information regarding your Internet service provider, and the frequency with which you visit various parts of this site. We use this information to manage the Website and make it more interesting and useful to you. We may offer public commenting through this site. If you use our commenting system and disclose personally identifiable information, you should be aware that such information could be read, collected or used by other users. We are not responsible for the personally identifiable information you choose to submit through public forums.
        </p>
        <h3>
          What information do we collect?
        </h3>
        <p>
          When you register as a user of this site, we may collect your email address, name and address, telephone number, and payment information (e.g., credit card number and billing address). Additionally, when you visit this site, we automatically collect some non-personally identifiable computer information, such as the type of computer operating system (such as Windows XP or Mac OS) and web browser (such as Chrome or Internet Explorer) you are using, information regarding your Internet service provider, and the frequency with which you visit various parts of this site. We use this information to manage the Website and make it more interesting and useful to you. We may offer public commenting through this site. If you use our commenting system and disclose personally identifiable information, you should be aware that such information could be read, collected or used by other users. We are not responsible for the personally identifiable information you choose to submit through public forums.
        </p>
      </div>
    </div>
    <footer>
      <ul>
        <li>
          <a
            class="link"
            href="/about"
          >
            About
          </a>
        </li>
        <li>
          <a
            class="link"
            href="/subscribe"
          >
            Subscribe
          </a>
        </li>
        <li>
          <a
            class="link"
            href="/privacy"
          >
            Privacy
          </a>
        </li>
      </ul>
    </footer>
  </div>
</div>
`;
